docker/tutorial-test.sh

```diff
@@ -28,13 +28,13 @@ spack mirror add tutorial /mirror
 spack gpg trust /mirror/public.key
 spack install zlib %clang
 spack install zlib @1.2.8
-spack install zlib %gcc@4.8
+spack install zlib %gcc@7.5.0
 spack install zlib @1.2.8 cppflags=-O3
 spack find
 spack find -lf
 spack install tcl
 spack install tcl ^zlib @1.2.8 %clang
-spack install tcl ^/64mn 
+spack install tcl ^/64mn
```

Comment:
In the most recent version of the tutorial this was `/hmvj`. It needs to be the hash of the `zlib@1.2.8 cppflags=-O3` spec.

Reply:
Ok, thanks for the heads up, I'll see what the hash is now and set it to that.